### PR TITLE
Fix #2454, add webp to supported image formats

### LINKED ISSUE #2454: Add webp support
Now that we've dropped IE11 support, webp should go on our list of supported formats!

### PR DESCRIPTION
**Pull request by BeebBenjamin**
This PR adds webp to the default image formats supported.

### CODE REVIEW

File: src/openseadragon.js [hunk 1]
```diff
@@ -2607,7 +2607,8 @@ function OpenSeadragon( options ){
          *      jpg:  true,
          *      png:  true,
          *      tif:  false,
-         *      wdp:  false
+         *      wdp:  false,
+         *      webp: true
```
Comment (iangilman) [inline src/openseadragon.js:2611]: Just a couple lines down from here there's an example that shows setting webp as supported; now that it is supported by default maybe we should change that example to something less likely to be supported by default, like bmp.

**Review by iangilman — CHANGES_REQUESTED** [review]
Awesome... Thank you for doing this. Would you mind just a minor update to the example below?

File: src/openseadragon.js (hunk 1 above)
Comment (BeebBenjamin) [reply src/openseadragon.js:2611]: OK, I've pushed a commit for this change.

**Review by iangilman — APPROVED** [review]
Thank you!

**Pull request merged**